src/components/tags/index.less

```diff
@@ -1,5 +1,14 @@
 @import '../styleguide/variables.less';
```

Comment:
It's a cool idea you have to show tags list full-width for mobile devices 💪.
And BTW, there is a way to simplify the implementation:
1. Add this to `src/components/article-card/index.less`
```
.article-card--tags {
  margin-left: 0;
  width: ~"calc(50% - 15px)";
  @media screen and (max-width: @screen-sm-max) { 
    width: 100%;
  }
}
```

2. Revert the changes you've made to `.tags` in `src/components/tags/index.less`
3. Add `article-card article-card--tags` classnames back to `src/templates/listPage.js `